communitiesuk/funding-service-design-fund-store
# FS-3235: Update fund store model for PDF generation

### Change description

- Add various properties to round/fund model for PDF display
- Per round we determine whether we display the logo on the PDF
- Per fund we add a logo and the some metadata about the owner organisation (DLUHC for now)

---



### How to test

See https://github.com/communitiesuk/funding-service-design-assessment/pull/383/
